charactersheet/charactersheet/init.js

```diff
@@ -24,6 +24,14 @@ var init = function(viewModel) {
     // Run migration
     PersistenceService.migrate(Migrations.scripts, Settings.version);
 
+    // Clipboard initialization.
+    var clipboard = new Clipboard('.btn');
+
+    clipboard.on('success', function(e) {
```

Comment:
Does this prohibit other uses for auto-copying? Like if we want to do it for other things?